src/web/mjs/connectors/Tapas.mjs

```diff
@@ -47,24 +47,34 @@ export default class Tapas extends Connector {
 
     async _getChapters(manga) {
         let chapterList = [];
+
         let request = new Request(new URL(manga.id, this.url), this.requestOptions);
-        let data = await this.fetchDOM(request, 'div.paging a.paging__button--num');
-        let pageCount = data.length === 0 ? 1 : parseInt(data.pop().href.match(/pageNumber=(\d+)/)[1]);
-        for(let page = 1; page <= pageCount; page++) {
-            let chapters = await this._getChaptersFromPage(manga, page);
-            chapterList.push(...chapters);
+        let data = await this.fetchDOM(request, 'meta[name="twitter:app:url:iphone"]');
+        const series_id = data[0].content.match(/((?:\/series\/)\d+)/)[0];
+
+        let has_next = true;
+        const time = Math.round(new Date().getTime());
+        let page = 1;
+        while (has_next) {
+            request = new Request(new URL(series_id+'/episodes?page='+page+'&sort=OLDEST&init_load=0&since='+time+'&large=true&last_access=0&', this.url), this.requestOptions);
```

Comment:
May use search params to compose URL
```javascript
let uri = new URL(series_id, this.url);
uri.searchParams.set('page', page);
uri.searchParams.set('sort', 'OLDEST'); // is this required?
uri.searchParams.set('init_load', 0);
uri.searchParams.set('since', time);
uri.searchParams.set('large', true); // is this required?
uri.searchParams.set('last_access', 0); // is this required?

let request = new Request(uri, this.requestOptions);
```

Reply:
Parameters are copied from a real query and I want to leave as little clues the query is not from a normal browser. Since the most stuff is static I opted for 'just a string' here.